Worker.Extension.Sql/src/SqlInputAttribute.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extension.Sql
+{
+    public sealed class SqlInputAttribute : InputBindingAttribute
+    {
+        /// <summary>
+        /// Constructs a new instance.
+        /// </summary>
+        /// <param name="commandText">The text of the command.</param>
+        /// <param name="commandType">Specifies whether commandText refers to a stored procedure or SQL query string.</param>
+        /// <param name="connectionStringSetting">The name of the app setting where the SQL connection string is stored.</param>
+        public SqlInputAttribute(string commandText, System.Data.CommandType commandType, string connectionStringSetting)
+        {
+            this.CommandText = commandText;
+            this.CommandType = commandType;
+            this.ConnectionStringSetting = connectionStringSetting;
+        }
+
+        /// <summary>
+        /// The name of the app setting where the SQL connection string is stored
+        /// (see https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnection?view=sqlclient-dotnet-core-2.0).
+        /// The attributes specified in the connection string are listed here
+        /// https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqlconnection.connectionstring?view=sqlclient-dotnet-core-2.0
+        /// For example, to create a connection to the "TestDB" located at the URL "test.database.windows.net" using a User ID and password,
+        /// create a ConnectionStringSetting with a name like SqlServerAuthentication. The value of the SqlServerAuthentication app setting
+        /// would look like "Data Source=test.database.windows.net;Database=TestDB;User ID={userid};Password={password}".
+        /// </summary>
+        public string ConnectionStringSetting { get; set; }
+
+        /// <summary>
+        /// For an input binding, either a SQL query or stored procedure that will be run in the database referred to in the ConnectionString.
```

Comment:
This is specifically an input binding so it seems unnecessary to call out specifically

(make sure you check all the other doc descriptions for the same kind of thing)